src/Randomizer.Shared/Enums/LocationStatus.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Randomizer.Shared.Enums
+{
+    /// <summary>
+    /// Indicates the current tracking status of the location
+    /// </summary>
+    public enum LocationStatus
```

Comment:
This enum should probably have a reasonable "zero" value. "Cleared" being the default seems a little... wrong, somehow.